bot/exts/info/information.py

```diff
@@ -153,7 +152,9 @@ async def server_info(self, ctx: Context) -> None:
         channel_counts = self.get_channel_type_counts(ctx.guild)
 
         # How many of each user status?
-        statuses = Counter(member.status for member in ctx.guild.members)
+        py_invite = await self.bot.fetch_invite("python")
+        online_presences = py_invite.approximate_presence_count
+        offline_presences = ctx.guild.member_count - online_presences
```

Comment:
Since the `!server` cmd always fetches the main guild invite, the presence counts will be very off elsewhere.

![image](https://user-images.githubusercontent.com/44734341/94991567-584a0500-0584-11eb-8181-d4e82100c68e.png)

Do you think there's something we could do about this, or is it not worth worrying about? It would be nice to also use the prod guild for the `member_count`, but then devbots cannot fetch it because they're not in it.

Reply:
Maybe it could just say, e.g.:
> :green_circle: Approximately `23,687` members online

Reply:
We could iterate through guild invites to generate that section.

Reply:
Actually, to expand on that we could iterate on *startup* to find a permanent invite..

Reply:
I don't see sending yet another request to the API just for some stats a nice thing to do.

We can instead make the invite link a config variable and use both approx presence count and approx member count rather than currently having hardcoded invite link, approx presence count and actual server size.

However as we grow bigger, approx member count will be more out of sync with reality I'd assume, however if we need such modularity it sounds like a good trade.

Reply:
> use both approx presence count and approx member count

Yeah, I wasn't aware this was possible, I assumed the current solution uses the `ctx` guild because it cannot get the info from the invite. I'm in favour of doing this then, as long as the invite gives us all we need.

Reply:
With 397d29a it should now display stats for the configurated invite link.